src/Nox.Types.EntityFramework/Types/Email/EmailConverter.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Nox.Types.EntityFramework.Types;
+
+/// <summary>
+/// The email converter.
+/// </summary>
+public class EmailConverter : ValueObjectConverter<Email, string>
```

Comment:
Since ValueObjectConverter has been deleted we need to change the implementation of this converter to inherit from EFs ValueConverter.